lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -772,6 +772,65 @@ def accessible_objects(user_or_attribute, operation, options = {})
       end
     end
 
+    # Version of accessible_objects which only returns objects that are
+    # ancestors of a specified root object or the object itself
+    def accessible_ancestor_objects(user_or_attribute, operation, root_object, options = {})
+      # Drop down to the root's stored policy element if not already there
```

Comment:
I don't think the meaning of "drop down" is immediately obvious to someone not already intimately familiar with all of the pm. Can we reword this?